x-pack/plugins/apm/public/components/shared/FilterBar/DatePicker.tsx

```diff
@@ -39,25 +39,10 @@ class DatePickerComponent extends React.Component<Props> {
 
   public componentDidMount() {
     this.dispatchTimeRangeUpdate();
```

Comment:
Cool, not sure if I follow yet. Are you saying other parts of the app should be able to control the date via URL? Because we already do that in the charts for the drag to select a time range, and it all works -- are you thinking that because of that, the url manipulation shouldn't happen in our date picker wrapper?

Reply:
>  Are you saying other parts of the app should be able to control the date via URL

I'm saying that the rest of the application should be able to work properly and read the url, without the date picker. 
Right now, if you removed the date picker the rest of the app wouldn't work, right?

> Because we already do that in the charts for the drag to select a time range, and it all works 

The chart manipulates the url (I hope). So should the date picker. Only exception is the refresh button, which should invoke a refresh action in redux.